src/Tizen.Nlp/Tizen.Nlp/NaturalLanguageProcess.cs

```diff
@@ -276,8 +315,14 @@ public Task<PosTagResult> PosTagAsync(string sentence)
         /// <param name="sentence">A sentence need to be processed.</param>
         /// <returns>NamedEntityRecognitionResult</returns>
         /// <since_tizen> 5 </since_tizen>
+        /// <feature>
+        /// http://tizen.org/feature/nlp
+        /// </feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
         public Task<NamedEntityRecognitionResult> NamedEntityRecognitionAsync(string sentence)
         {
+            ValidateFeatureSupported(FeatureNlp);
```

Comment:
No need to check feature in every method, becuase It can't called if not created. 
and you already check in constructor